python/py-spdx-lookup/Portfile

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-spdx-lookup
+version             0.3.2
+revision            0
+
+maintainers         {@korusuke somaiya.edu:karan.sheth} openmaintainer
+description         SPDX license list query tool
+long_description    ${description}
+
+platforms           darwin
+supported_archs     noarch
+
+homepage            https://github.com/bbqsrc/spdx-lookup-python
+master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
+distname            ${python.rootname}-${version}
+
+license             BSD-2-clause
```

Comment:
I think this should just be ```BSD``` here and in the the ```py-upt-*``` ports. As far as I can tell MacPorts doesn't differentiate between 2/3-clause version. @mojca?

Reply:
I checked this [page](https://trac.macports.org/wiki/PortfileRecipes#licensekeyword) as suggested by @mojca earlier  and in that page, there's only `BSD` and `BSD-old` but seeing that the page was not updated for a long time I tried running the command given there and in that following BSD clause are there :

* BSD
* BSD-2-clause
* BSD-3-clause
* BSD-old

For now, I have changed it to BSD as you suggested.

Reply:
True, but you probably run the command on your PR branch, which is why you got the BSD-3-Clause. The BSD-2-Clause is indeed present in master as well, but only for one other port (if I checked this correctly).

Reply:
Oh ok, right my bad!
I have made the changes suggested by you.
Is there anything left?